template/src/screens/__tests__/demoSlice.test.ts

```diff
@@ -58,47 +26,5 @@ describe('DemoSlice', () => {
 
       expect(result.counter).toEqual(410)
     })
-
-    it('set getLatestComicAsync to Loading if there is no comic in store', () => {
-      const result = demoSlice.reducer(initialState, {
-        type: getLatestComicAsync,
-        payload: comicMockResponse,
-      })
-
-      expect(result.comic).toEqual(Loading)
-    })
-
-    it('set getLatestComicAsync to Refreshing if there is already comic in store', () => {
```

Comment:
This test is removed, because it's testing switching to `Refreshing` state when request is started. It's handled by lib now, which is tested separately (in lib).